src/pages/Header.tsx

```diff
@@ -286,16 +294,92 @@ class Header extends React.Component<Props, State> {
         </ReactResizeDetector>
         <Portal container={drawerContainer}>
           <Drawer
-            toggleDrawer={this.toggleDrawer}
-            drawerOpen={this.state.drawerOpen}
+            isFeedsManagerFeatureEnabled={isFeedsManagerFeatureEnabled}
+            toggleDrawer={toggleDrawer}
+            drawerOpen={drawerOpen}
             authenticated={authenticated}
             submitSignOut={submitSignOut}
           />
         </Portal>
       </AppBar>
     )
-  }
-}
+  },
+)
+
+// class Header extends React.Component<Props, State> {
```

Comment:
Should we just delete this? Can always recover it later in git

Reply:
Sorry. Yeah it was meant to be deleted